src/app/organizations/manage/policy-edit.component.html

```diff
@@ -153,19 +156,32 @@ <h3 class="mt-4">{{'passphrase' | i18n}}</h3>
                 <ng-container *ngIf="type === policyType.SendOptions">
                     <h3 class="mt-4">{{'options' | i18n}}</h3>
                     <div class="form-check">
-                        <input class="form-check-input" type="checkbox" id="sendDisableHideEmail" [(ngModel)]="sendDisableHideEmail"
-                            name="SendDisableHideEmail">
+                        <input class="form-check-input" type="checkbox" id="sendDisableHideEmail"
+                            [(ngModel)]="sendDisableHideEmail" name="SendDisableHideEmail">
                         <label class="form-check-label" for="sendDisableHideEmail">{{'disableHideEmail' | i18n}}</label>
                     </div>
                 </ng-container>
+                <ng-container *ngIf="type === policyType.ResetPassword">
+                    <h3 class="mt-4">{{'resetPasswordPolicyAutoEnroll' | i18n}}</h3>
+                    <p>{{'resetPasswordPolicyAutoEnrollDescription' | i18n}}</p>
+                    <app-callout type="warning">
+                        {{'resetPasswordPolicyAutoEnrollWarning' | i18n}}
+                    </app-callout>
+                    <div class="form-check">
+                        <input class="form-check-input" type="checkbox" id="autoEnrollEnabled"
+                            [(ngModel)]="resetPasswordAutoEnroll" name="AutoEnrollEnabled">
+                        <label class="form-check-label"
+                            for="autoEnrollEnabled">{{'resetPasswordPolicyAutoEnrollCheckbox' | i18n }}</label>
+                    </div>
+                </ng-container>
             </div>
             <div class="modal-footer">
                 <button type="submit" class="btn btn-primary btn-submit" [disabled]="form.loading">
                     <i class="fa fa-spinner fa-spin" title="{{'loading' | i18n}}" aria-hidden="true"></i>
                     <span>{{'save' | i18n}}</span>
                 </button>
-                <button type="button" class="btn btn-outline-secondary"
-                    data-dismiss="modal">{{'cancel' | i18n}}</button>
+                <button type="button" class="btn btn-outline-secondary" data-dismiss="modal">{{'cancel' |
+                    i18n}}</button>
```

Comment:
Odd place to wrap the button text binding

Reply:
Agreed - updated.